src/main/scala/org/broadinstitute/hail/check/Gen.scala

```diff
@@ -20,6 +20,16 @@ case class Parameters(rng: RandomDataGenerator, size: Int, count: Int) {
 }
 
 object Gen {
+  def getRng(): Gen[RandomDataGenerator] = Gen { p => p.rng }
+
+  def randomSquareOfAreaAtMost(a: Int): Gen[(Int, Int)] = Gen { p =>
```

Comment:
Random seems redundant since it is returning a generator.  What about `squareOfAreaAtMostSize: Gen[(Int, Int)]` and then use `resize` below?